CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Review/ReviewViewController.swift

```diff
@@ -371,28 +376,31 @@ extension ReviewViewController {
         buttonLeadingConstraint.priority = UILayoutPriority.defaultLow
 
         NSLayoutConstraint.activate([
-            tipLabel.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 16),
+            tipLabel.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: Constants.padding),
             tipLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor),
             tipLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tipLabel.heightAnchor.constraint(equalToConstant: Constants.titleHeight),
 
-            collectionView.topAnchor.constraint(equalTo: tipLabel.bottomAnchor, constant: 16),
+            collectionView.topAnchor.constraint(equalTo: tipLabel.bottomAnchor, constant: Constants.padding),
             collectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
             collectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            collectionViewHeightConstraint,
 
-            pageControl.topAnchor.constraint(equalTo: collectionView.bottomAnchor, constant: 32),
+            pageControl.topAnchor.constraint(equalTo: collectionView.bottomAnchor, constant: Constants.padding * 2),
             pageControl.leadingAnchor.constraint(equalTo: view.leadingAnchor),
             pageControl.trailingAnchor.constraint(equalTo: view.trailingAnchor),
 
-            processButton.topAnchor.constraint(equalTo: pageControl.bottomAnchor, constant: 32),
-            processButton.widthAnchor.constraint(equalToConstant: 204),
+            processButton.topAnchor.constraint(equalTo: pageControl.bottomAnchor, constant: Constants.padding * 2),
```

Comment:
[Suggestion] Could you please refactor this method and extract some code into specific layout methods?